package/utils.py

```diff
@@ -66,7 +67,7 @@ def normalize_license(license):
     """
     if license is None:
         return "UNKNOWN"
-    if license.strip() in settings.LICENSES:
+    if license.strip() in deprecated_classifiers:
```

Comment:
See above change too: 

```suggestion
    if license.strip() in classifiers:
```